design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,47 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task, it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review, but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview / Problem Statement
```

Comment:
Could this be replaced with a link back to the task? The task should have a title and description and be the source of truth for describing the work.

Reply:
Agreed - we should include a link to the source task in whichever repo/tool they are using.

Reply:
I have added the flexible option to link to the task so we can have the details even if the link is unavailable.